packages/elements/src/types/element.ts

```diff
@@ -40,6 +46,8 @@ export interface Mask {
   name?: string;
   path?: string;
   ratio?: number;
+  iconPath?: string;
+  iconRatio?: number;
   supportsBorder?: boolean;
 }
```

Comment:
Wondering: should the `Mask` type better live in the `masks` package?

Reply:
The `masks` package depends on the `elements` package, and that dependency is pretty locked-in. There are all the components for display, frame, and output, that have element typings as well as several util functions (`shouldDisplayBorder`, `canMaskHaveBorder`, ...). And the big one is the `BorderedMaskedElement` using many element features.

Some of these could be solved by utilizing `DimensionableElement` from units package (though the border in that package does not include a color which masks need, which would add a dependency on the patterns package, but that's probably okay), others could be solved by also adding a `MaskableElement` to the masks package, but we still also need things like the `flip` and `isBackground` properties from the elements (in the `BorderedMaskedElement`), so that would require making even more element interfaces.

It's doable, but it's getting a bit complex (a naming is getting terrible 😂). Let's move this idea to a new ticket.